include/IGPUBuffer.h

```diff
@@ -6,7 +6,7 @@
 #ifndef __I_GPU_BUFFER_H_INCLUDED__
 #define __I_GPU_BUFFER_H_INCLUDED__
 
-#include "irr/core/IBuffer.h"
+#include "irr/asset/IBuffer.h"

```

Comment:
IBuffer is staying in `core::`

Reply:
i cant stay in core because it has to derive from IDescriptor